spec.md

```diff
@@ -2274,6 +2374,20 @@ message VolumeUsage {
   // Units by which values are measured. This field is REQUIRED.
   Unit unit = 4;
 }
+
+// VolumeCondition represents the current health condition of a volume
+message VolumeCondition {
```

Comment:
nit: What was wrong with `VolumeHealth` again? `VolumeCondition` seems weird to me.

Reply:
I think it is because we were using Conditions (condition_true, condition_false, condition_unknown) here in the previous version of the PR.  VolumeHealth condition sounds weird.  So James proposed to rename to VolumeCondition.

Now that we use .google.protobuf.BoolValue.  Maybe we should rename back to VolumeHealth?

@jdef what do you think?

Reply:
Changed from .google.protobuf.BoolValue to bool as "abnormal" is required now.

Reply:
s/health condition/condition/ everywhere here and I'm happy with it. "health condition" is too awkward. Just get rid of the word "health" everywhere please.

What does healthy mean? I think it means different things to different people (as our many meetings seem to suggest). "Volume condition" - people actually have to read the docs to understand what this means. I like that. And it forces us to write decent documentation for "condition" fields.

"Indicates that the condition of a volume is not normal." -- this could use some polishing, it's too sparse. We don't need a novel here, but we need *something* more.  Maybe something along the lines of:

> *Normal* volumes are online (i.e. available for use) and operating optimally. An *abnormal* volume does not meet these criteria.

Reply:
"online" means volume is attached, but we also look at volumes that are provisioned but not attached yet.  So I changed to the following:
  
  // Normal volumes are available for use and operating optimally.
  // An abnormal volume does not meet these criteria.